src/cascadia/TerminalApp/JsonUtils.h

```diff
@@ -269,10 +271,25 @@ namespace TerminalApp::JsonUtils
         }
     };
 
-    // (GUID and winrt::guid are mutually convertible!)
+    // GUID and winrt::guid are mutually convertible,
+    // but IReference<winrt::guid> throws some of this off
     template<>
-    struct ConversionTrait<winrt::guid> : public ConversionTrait<GUID>
+    struct ConversionTrait<winrt::guid>
     {
+        winrt::guid FromJson(const Json::Value& json) const
+        {
+            return static_cast<winrt::guid>(ConversionTrait<GUID>{}.FromJson(json));
+        }
+
+        bool CanConvert(const Json::Value& json) const
+        {
+            return ConversionTrait<GUID>{}.CanConvert(json);
+        }
+
+        std::string TypeDescription() const
+        {
+            return "guid";

```

Comment:
prefer `ConversionTrait<GUID>{}.TypeDescription()` to reduce number of times we have the string `"guid"` in code